plugin/surround.vim

```diff
@@ -8,6 +8,9 @@ if exists("g:loaded_surround") || &cp || v:version < 700
 endif
 let g:loaded_surround = 1
 
+let g:insert_space_left = get(g:, 'surround_insert_space_left', 1)
+let g:insert_space_right = get(g:, 'surround_insert_space_right', 0)
```

Comment:
Looks like this should be `s:insert_space_right`. No point in defining a second global config variable.